lisa/analysis/rta.py

```diff
@@ -269,7 +269,7 @@ def __missing__(self, key):
                    raise ValueError(f'Unexpected phase number "{key}"')
 
             def f(df):
-                pid, comm = df.name
```

Comment:
This case is "interestingly" not affected. Pandas manages to set this attribute from `DataFrame.apply()` in a "special" way that will not set a column named "name". This means that in that specific case, `df.name != df['name']`.